### PR TITLE
build: Do not specialize source tree for API variants

### PR DESCRIPTION
**Pull request by aqnuep**
In order to avoid having to deal with mixed generated vs manual sources / headers in this repo, we decided to not maintain separate source trees at all for API variants downstream, so this change removes the existing path parameterizations based on the target API variant.

### CODE REVIEW

**Review by spencer-lunarg — APPROVED** [review]
cc @charles-lunarg

**Comment by charles-lunarg** [comment]
@aqnuep Is this ready to merge? (I'm unsure if you have write access so want to double check that you aren't delaying merging for some reason)

**Comment by aqnuep** [comment]
Yes, it should be ready, I was only waiting for your approval.

**Pull request merged**
